### DISCUSSION TITLE
posts/2022/07/bundling-up-rebuilding-my-hugo-site/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#43] [Announcements]
# posts/2022/07/bundling-up-rebuilding-my-hugo-site/

O ye new users, run as fast as you can — away from the way I was managing my content, and toward using page bundles, as I’m doing now.

https://www.brycewray.com/posts/2022/07/bundling-up-rebuilding-my-hugo-site/

**Comment by simplicitywebdesign** [comment]
How do you then account for having huge content directories full of images? As far as I can tell, Hugo is exceptional for text blogs, but as soon as you start adding lots of photos or other large documents, you have to start screwing around with Git LFS and that's just more headache.

I feel Hugo will always be stuck in this category of requiring someone to completely tool up just to get a basic site going. I was expecting a steep learning curve but so far images are just absolutely killing me. I've gone back to Gulp and hosting remotely. Hugo's image processing just isn't worth the hassle.

**Reply by brycewray** [reply]
@simplicitywebdesign Haven’t ever hosted “lots of” such items so truthfully can’t give you much of an answer. However, regarding this part of your comment:

> I’ve gone back to Gulp and hosting remotely.

. . . could you clarify? By “hosting remotely,” do you mean having your images served by Cloudinary or another service of that nature? If so, that’s a perfectly acceptable substitute for a site with numerous, large image files (as long as you monitor your usage to avoid any overage issues, of course), and I can recommend it. As for Gulp: I guess it depends on one’s workflow. 😄
